videojs/http-streaming
# feat: enable caption positioning

## Description
This consumes the work completed in mux.js here: https://github.com/videojs/mux.js/pull/434

The goal of this work is to ensure that we can position CAE 608 captions. The bulk of the work was done in mux.js, but this just ensures that we take that data and turn them into valid `VTTCue`s . This will ensure that the captions are positioned correctly based on that CAE 608 caption data.

## To Test
Pull down this branch and run `npm i` and `npm start`, and then navigate to http://localhost:9999. In the closed caption menu, choose the CC1 option and see that the captions are positioned in different places on the screen.

## Specific Changes proposed
- Package updates to mux.js and video.js
- Ensuring we add the correct properties to VTTCues per caption sent to VHS.
- Tests regarding the above

## Requirements Checklist
- [x] If necessary, more likely in a feature request than a bug fix
  - [x] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://jsbin.com/gejugat/edit?html,output))
- [ ] Reviewed by Two Core Contributors
